django_beanstalkd/decorators.py

```diff
@@ -172,3 +172,120 @@ def __call__(instance, pk_str):
                         raven_client.captureException()
 
         return wrapper()
+
+
+class retry_data_beanstalk_job(object):
```

Comment:
I did consider updating the existing one. But I was feeling lazy, and didn't want to have to find and update other uses, test, etc.